ksql-serde/src/main/java/io/confluent/ksql/serde/connect/ConnectDataTranslator.java

```diff
@@ -29,6 +28,11 @@
 import org.apache.kafka.connect.data.Timestamp;
 import org.apache.kafka.connect.errors.DataException;
 
+/**
+ * Translates full set of Connect types to the limited subset supported by KSQL.
```

Comment:
This class also takes care of case-sensitivity.

Reply:
Done in main PR. (Sorry, forgot to address before merging!)